src/backend/distributed/sql/citus--9.4-3--9.4-2.sql

```diff
@@ -0,0 +1,20 @@
+-- citus--9.4-3--9.4-2
+-- this is a downgrade path that will revert the changes made in citus--9.4-2--9.4-3.sql
+-- 9.4-2--9.4-3 was added later as a patch to improve master_update_table_statistics
```

Comment:
Minor: I think here we could mention that we have this downgrade script so that we can continue from the main upgrade path when upgrading to later versions.